pkg/apis/serving/v1/revision_types.go

```diff
@@ -135,6 +135,13 @@ type RevisionStatus struct {
 	// may be empty if the image comes from a registry listed to skip resolution.
 	// +optional
 	ImageDigest string `json:"imageDigest,omitempty"`
+	// ImageDigests holds the resolved digest for the image specified
+	// within .Spec.Container.Image. The digest is resolved during the creation
+	// of Revision. This will be filled if there are multiple container specified.
+	// ImageDigests holds the digest for all the .Spec.Container.Image which are non serving.
```

Comment:
We should put all images in this map -- not just the non-serving images.

Reply:
So do you mean

Whether if its single container or multiple container we need to fill `ImageDigests`  and `ImageDigest` for backward compatibility 
ex:
1. Single container
**ImageDigest:**  `digestValue`
**ImageDigests:** 
    `containerName`: `digestValue`

2. Multiple container (consider 2 container) 
**ImageDigest:**  `digestValue`
**ImageDigests:** 
     `servingContainerName`: `digestValue`
    `nonServingContainerName`: `digestValue`